extensions/amp-story/1.0/story-analytics.js

```diff
@@ -152,6 +156,10 @@ export class StoryAnalyticsService {
       details.repeated = true;
     }
 
+    if (element) {
+      details.tagName = element.tagName.toLowerCase();
```

Comment:
Since we are adding more element related details, `pageDetails` is no longer a good name to me : ) maybe `eventDetails`? 
Not blocking.